terraform/modules/azure/review-app.tf

```diff
@@ -0,0 +1,6 @@
+resource "azurerm_resource_group" "review_app_rg" {
+  count    = var.create_env_resource_group ? 1 : 0
```

Comment:
As discussed we can create a resource group for any environment, so the conditional can be removed

Reply:
Sorted